plugin/net/driver.go

```diff
@@ -206,7 +211,7 @@ func (driver *driver) DiscoverDelete(disco *api.DiscoveryNotification) error {
 }
 
 func vethPair(id string) (string, string) {
-	return "vethwl" + id[:5], "vethwg" + id[:5]
+	return "vethwl" + id[:8], "vethwg" + id[:8]
```

Comment:
Do we know for sure that the id will be at least 8 (or, previously, 5) long?

In any case, if we made this safe for shorted ids then I think this would then allow us to avoid the leak on upgrade by attempting to delete `vethPair(id[:5])` if the delete of `vethPair(id)` fails.